node/src/reactor/joiner.rs

```diff
@@ -728,6 +728,22 @@ impl reactor::Reactor for Reactor {
                         event_stream_server::Event::BlockFinalized(block),
                     ),
                 ),
+                ConsensusAnnouncement::Equivocation {
+                    era_id,
+                    public_key,
+                    timestamp,
+                } => reactor::wrap_effects(
+                    Event::EventStreamServer,
+                    self.event_stream_server.handle_event(
+                        effect_builder,
+                        rng,
+                        event_stream_server::Event::Equivocation {
+                            era_id,
+                            public_key: *public_key,
+                            timestamp,
+                        },
+                    ),
+                ),
```

Comment:
(Not sure I understand the difference to what the `Validator` reactor does.)

Reply:
We're not consistent between the two reactors in how we handle this.  The validator recurses back into `dispatch_event` in order to achieve what's being spelled out here, i.e. calling `wrap_effects`.  There's no real difference really.

Hopefully once Marc's had a chance to complete his work on the macros for this we won't have to worry about such things!